kayenta-core/src/main/java/com/netflix/kayenta/canary/CanaryExecutionStatusResponse.java

```diff
@@ -74,6 +80,12 @@
   protected String endTimeIso;
 
   // If set, these are the account names used for this run.
+
   protected String storageAccountName;
+
+  public String getStorageAccountName() {
```

Comment:
Same question as up above re lombok. I thought `@Data` meant `@Getter` is implicitly applied. Maybe it doesn't work on `protected` fields? If so, can just add `@Getter` to the needed fields, no?